libs/parallelism/executors/include/hpx/executors/dataflow.hpp

```diff
@@ -26,11 +26,13 @@
 #include <hpx/futures/traits/future_access.hpp>
 #include <hpx/futures/traits/is_future.hpp>
 #include <hpx/modules/memory.hpp>
-#include <hpx/modules/naming.hpp>
 #include <hpx/pack_traversal/pack_traversal_async.hpp>
 #include <hpx/threading_base/annotated_function.hpp>
 #include <hpx/threading_base/thread_num_tss.hpp>
 #include <hpx/type_support/always_void.hpp>
+#if defined(HPX_HAVE_DISTRIBUTED_RUNTIME)
+#include <hpx/modules/naming.hpp>
+#endif
```

Comment:
I know this dependency was already there, but is there something we can do about it already now?

Reply:
Let's leave this alone for now as it will involve (major) unrelated restructurings.